lib/common/zstd_internal.h

```diff
@@ -24,6 +24,8 @@
 #include "mem.h"
 #include "debug.h"                 /* assert, DEBUGLOG, RAWLOG, g_debuglevel */
 #include "error_private.h"
+#define ZSTD_DEPS_NEED_MALLOC
```

Comment:
@Cyan4973 - Is there any issue with adding this to `zstd_internal.h`?

Reply:
Well, `zstd_internal.h` is used everywhere within `libzstd`, so this change can impact a lot of places.
I'm not sure if I can understand all the implications.